### ISSUE TITLE
Feature Request : Add the ability to use only custom check

### ISSUE THREAD

**Issue by soufiane444** [bridgecrewio/checkov#722]
I want the ability to use only custom check.
Currently with the  --external-checks-dir options , checkov load also the common the built-in policies

**Comment by christophetd** [comment]
This is definitely needed! It could also be implemented using a CLI argument `--skip-check-pattern` which could be used as follows:

```
checkov -d . --external-checks-dir /path/to/checks --skip-check-pattern 'CKV_*'
```

**Comment by libertyy** [comment]
Hi! This ticket is already filed at 
https://github.com/bridgecrewio/checkov/issues/563

**Comment by libertyy** [comment]
TBH, I've been waiting to see how https://github.com/bridgecrewio/checkov/issues/560 shakes out before iterating on my own issue/P.R.

**Comment by soufiane444** [comment]
I created the pul request : https://github.com/bridgecrewio/checkov/pull/749

**Comment by metahertz** [comment]
fixed by #750. Closing 👍  Thanks for your contributions @soufiane444!! 👑
